### DISCUSSION TITLE
How to import data from paid PA to a selfhosted version?

### DISCUSSION THREAD

**Discussion by Pryla** [plausible/analytics#1932] [Self-Hosted Support] [answered]
Hi guys,

I have spent much money on your hosted version but know I started to go down the rabbit hole of selfhosting. 

Everybody of this community talks about importing data from google analytics but neither in the docs, the issue page nor here inside the discussion area is anything mentioned about importing plausible analytics data back into plausible.

I am sorry if I have missed it somewhere on the roadmap. Maybe someone could clarify.

Thank you :)

**Comment by metmarkosaric** [comment, marked as answer]
hi @Pryla! currently there's no easy import tool available to go from self-hosted to cloud or vice versa unfortunately. we'd like to build it but we're a small team with limited resources and this is a major undertaking so things will take time. i cannot make any promise on when we'll be able to work on this as there are some other more frequently requested features we're working on at the moment. 

there's a thread for all the import / export topics here: https://github.com/plausible/analytics/discussions/211. until now, we've managed to build an export tool for both the cloud and self-hosted which was the most popular request. we're recently introduced GA import tool on the cloud and will work on introducing GA import tool in the next version of self-hosted as this is another popular request. 

next up on the export / import topic in terms of popularity of requests are are a more general import tool that supports other analytics tools, GA4 import and a tool that allows data transfers between self-hosted and cloud. we'll post any updates on this topic in the thread linked above. thanks for understanding!

**Comment by StacySiz** [comment]
Any updates on this?

**Reply by ruslandoga** [reply]
👋 @StacySiz 

There is some work in progress in #3692

**Reply by ruslandoga** [reply]
v2.1.0-rc.1 supports CSV exports and imports: https://github.com/plausible/analytics/discussions/4084
